pkg/controller/releasemanager/incarnation.go

```diff
@@ -566,6 +566,25 @@ func (i *IncarnationCollection) releasable() []Incarnation {
 	return r
 }
 
+func (i *IncarnationCollection) unreleasable() []Incarnation {
+	releasableTags := []string{}
+	for _, incarnation := range i.releasable() {
+		releasableTags = append(releasableTags, incarnation.tag)
+	}
+
+	unreleasable := []Incarnation{}
+Outer:
+	for _, incarnation := range i.sorted() {
+		for _, tag := range releasableTags {
+			if incarnation.tag == tag {
+				continue Outer
```

Comment:
fyi I own `ihatego.com`

Reply:
what's the big O of this function? 😂

Reply:
golang is trying to tell me to refactor using a map.

Reply:
_tfw you steal many great ideas from Python – but forget the `set` type_ 😐

Reply:
_we still wanted to capture some of the fundamental experience of writing C – misery_